content/en/docs/ops/common-problems/network-issues/index.md

```diff
@@ -233,7 +233,235 @@ server {
+
+{{< text yaml >}}
+spec:
+  ...
+  http:
+  - match: ...
+{{< /text >}}
+
+#### Gateway with TLS passthrough
+
+{{< text yaml >}}
+apiVersion: networking.istio.io/v1beta1
+kind: Gateway
+metadata:
+  name: gateway
+spec:
+  selector:
+    istio: ingressgateway
+  servers:
+  - hosts:
+    - "*"
+    port:
+      name: https
+      number: 443
+      protocol: HTTPS
+    tls:
+      mode: PASSTHROUGH
+---
+apiVersion: networking.istio.io/v1beta1
+kind: VirtualService
+metadata:
+  name: virtual-service
+spec:
+  gateways:
+  - gateway
+  hosts:
+  - httpbin.example.com
+  http:
+  - route:
+    - destination:
+        host: httpbin.org
+{{< /text >}}
+
+In this configuration, the virtual service is attempting to match HTTP traffic against TLS traffic passed through the gateway.
+This will result in the virtual service configuration having no effect. You can observe that the HTTP route is not applied using
+the `istioctl proxy-config listener` and `istioctl proxy-config route` commands.
+
+To fix this, you should switch the virtual service to configure `tls` routing:
+
+{{< text yaml >}}
+spec:
+  tls:
+  - match:
+    - sniHosts: ["httpbin.example.com"]
+    route:
+    - destination:
+        host: httpbin.org
+{{< /text >}}
+
+Alternatively, you could terminate TLS, rather than passing it through, by switching the `tls` configuration in the gateway:
+
+{{< text yaml >}}
+spec:
+  ...
+    tls:
+      credentialName: sds-credential
+      mode: SIMPLE
+{{< /text >}}
+
+### Double TLS (TLS origination for a TLS request) {#double-tls}
+
+When configuring Istio to perform {{< gloss >}}TLS origination{{< /gloss >}}, you need to make sure
+that the application sends plaintext requests to the sidecar, which will then originate the TLS.
+
+The following `DestinationRule` originates TLS for requests to the `httpbin.org` service,
+but the corresponding `ServiceEntry` defines the protocol as HTTPS on port 443.
+
+{{< text yaml >}}
+apiVersion: networking.istio.io/v1beta1
+kind: ServiceEntry
+metadata:
+  name: httpbin
+spec:
+  hosts:
+  - httpbin.org
+  ports:
+  - number: 443
+    name: https
+    protocol: HTTPS
+  resolution: DNS
+---
+apiVersion: networking.istio.io/v1beta1
+kind: DestinationRule
+metadata:
+  name: originate-tls
+spec:
+  host: httpbin.org
+  trafficPolicy:
+    tls:
+      mode: SIMPLE
+{{< /text >}}
+
+With this configuration, the sidecar expects the application to send TLS traffic on port 443
+(e.g., `curl https://httpbin.org`), but it will also perform TLS origination before forwarding requests.
+This will cause the reqeusts to be double encrypted.
+
+For example, sending a request like `curl https://httpbin.org` will result in an error:
+`(35) error:1408F10B:SSL routines:ssl3_get_record:wrong version number`.
+
+You can fix this example by changing the port protocol in the `ServiceEntry` to HTTP:
+
+{{< text yaml >}}
+spec:
+  hosts:
+  - httpbin.org
+  ports:
+  - number: 443
```

Comment:
We may want to switch this to
```
  ports:
  - number: 80
    name: http
    protocol: HTTP
    targetPort: 443
```

This allows the user applications to send requests to `http://httpbin.org/get` (note - port 80 not 443), rather than having to change their app.

I didn't do this originally since targetPort is new in 1.8

Reply:
Hmm, this is a very interesting point. If we change it, then I think I need to remove the next paragraph that explains there is a better way as described in https://preliminary.istio.io/latest/docs/tasks/traffic-management/egress/egress-tls-origination/#tls-origination-for-egress-traffic.

Maybe we also need to update that task to use this new approach too? WDYT?

Reply:
Fixed. TLS origination task should also be updated in a future PR.